.github/workflows/cicd.yml

```diff
@@ -0,0 +1,32 @@
+name: cicd
+
+on:
+  push:
+    branches:
+      - v3.0.0-openmldb
+  pull_request:
```

Comment:
add `workflow_dispatch` so we can run manually

Reply:
okay.